src/main/java/no/nav/bidrag/reisekostnad/model/forespørselExtensions.kt

```diff
@@ -3,4 +3,12 @@ package no.nav.bidrag.reisekostnad.model
 import no.nav.bidrag.reisekostnad.database.datamodell.Forespørsel
```

Comment:
Kan vi endre til forespørselUtvidelser? Vi bør vel også holde oss til samme navnestandard for .kt-filer som for klasser (kameltype, hvert ord starter med stor bokstav)?